css/selectors/focus-visible-script-focus-001.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>CSS Test (Selectors): Script focus without any previous user interaction matches :focus-visible</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/selectors-4/#the-focus-visible-pseudo" />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+<style>
+  #warning {
+    display: none;
+    background: red;
+  }
+
+  @supports not (selector(:focus-visible)) {
```

Comment:
FWIW I don't think you need the parenthesis around `selector(...)`, `@supports not selector(:focus-visible)` should work.

Reply:
Indeed that's not needed, but many tests use that. I think I'll change them all together after merging this.

Reply:
FWIW fixed in #28504.